setup.py

```diff
@@ -807,6 +818,8 @@ def detect_simple_extensions(self):
         # uses modf().
         self.add(Extension('_datetime', ['_datetimemodule.c'],
                            libraries=['m']))
+        # zoneinfo module
+        self.add(Extension('_czoneinfo', ['zoneinfomodule.c'])),
```

Comment:
I suggest to call the module _zoneinfo. I don't think that any other C extension of the stdlib have "c" in their name. But it's very common to have a very few functions implemented in C and get them from the Python code. I'm not talking about drop-in replacements modules like _datetime vs datetime.py. I'm thinking at _functools which only implements lru_cache.

Reply:
OK, done.